code/__HELPERS/game.dm

```diff
@@ -224,8 +224,10 @@
 		processing_list += T.contents // We can shave off one iteration by assuming turfs cannot hear
 	else  // A variation of get_hear inlined here to take advantage of the compiler's fastpath for obj/mob in view
 		var/lum = T.luminosity
+		processing_list = list()

```

Comment:
this is an empty list created just 5 lines above, this line will never do anything